app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] [OPTIONAL] Great job implementing devise for sign-up and sign-in, however, it will be nice if you enable `:confirmable` so that a user will be able to confirm their email after sign-up and before finally signing in. This helps to keep the account more secure.